dotnet/src/Connectors/Connectors.AzureOpenAI/Connectors.AzureOpenAI.csproj

```diff
@@ -31,5 +29,6 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\SemanticKernel.Core\SemanticKernel.Core.csproj" />
+    <ProjectReference Include="..\Connectors.OpenAIV2\Connectors.OpenAIV2.csproj" />
```

Comment:
Doesn't this mean we are also pulling in the open ai client nuget?

Reply:
Right, this reference will pull the open ai client. This same client pulled anyway by the azure open ai nuget.